libs/language-server/src/lib/meta-information/block-meta-inf.ts

```diff
@@ -15,21 +24,90 @@ interface BlockDocs {
   examples?: ExampleDoc[];
 }
 
-export abstract class BlockMetaInformation extends MetaInformation {
+export class BlockMetaInformation extends MetaInformation {
```

Comment:
This class basically became an AST wrapper (like Valuetype and others). I did not move it yet as I'd like to have the `MetaInformation` stuff cohesively in one place. Would move them as soon as we did the same for `Constraints`.